### PR TITLE
Andor install

### PR DESCRIPTION
**Pull request by wschoenell**
Adds `make install` to the andor optional ccd driver. It is needed so `indigo_server indigo_ccd_andor` works, otherwise the user has to copy the `.so` file manually.

### CODE REVIEW

File: indigo_optional_drivers/ccd_andor/Makefile [hunk 1]
```diff
@@ -2,13 +2,12 @@
 # Copyright (c) 2018 Rumen G. Bogdanovski
 # All rights reserved.
 
+include ../../Makefile.inc
+
 DRIVER_NAME=indigo_ccd_andor
 
 ROOT=../..
-BUILD_ROOT=$(ROOT)/build
```
Comment (rumengb) [inline indigo_optional_drivers/ccd_andor/Makefile:8]: Why did you remove those lines? This will break global make file.

File: indigo_optional_drivers/ccd_andor/Makefile [hunk 2]
```diff
@@ -42,5 +41,8 @@ $(BUILD_DRIVERS)/$(DRIVER_NAME): $(DRIVER_NAME)_main.o $(BUILD_DRIVERS)/$(DRIVER
 $(BUILD_DRIVERS)/$(DRIVER_NAME).$(SOEXT): $(DRIVER_NAME).o
 	$(CC) -shared -o $@ $^ $(LDFLAGS) -lindigo -landor
 
+install:
```
Comment (rumengb) [inline indigo_optional_drivers/ccd_andor/Makefile:44]: Why do you need install target? The driver when built is placed in the same directory with other indigo drivers and will be installed from the main Makefile.

**Review by rumengb — CHANGES_REQUESTED** [review]

File: indigo_optional_drivers/ccd_andor/Makefile (hunk 2 above)
Comment (wschoenell) [reply indigo_optional_drivers/ccd_andor/Makefile:44]: I added the `make install` because running it on the main Makefile 
```console
# cd indigo/ 
# make
# cd indigo_optional_drivers/ccd_andor/
# make
# cd ../../
# make install # now back at indigo/ main folder
```

then, run:

```
[root@5dac5c4dbb17 indigo]# indigo_server indigo_ccd_andor
05:03:20.322792 indigo_server: Driver indigo_ccd_andor can't be loaded (indigo_ccd_andor.so: cannot open shared object file: No such file or directory)
```

I get the error above. That's why I created the `make install` in this pull request. So this works:
```[root@5dac5c4dbb17 ccd_andor]# make install
install -m 0644 /indigo/build/drivers/indigo_ccd_andor.so //usr/local/lib
[root@5dac5c4dbb17 ccd_andor]# cd
[root@5dac5c4dbb17 ~]# indigo_server indigo_ccd_andor
```

Am I missing something to make it work without the necessity of this change?

File: indigo_optional_drivers/ccd_andor/Makefile (hunk 1 above)
Comment (wschoenell) [reply indigo_optional_drivers/ccd_andor/Makefile:8]: These lines were removed because I added the line:
`include ../../Makefile.inc`

`Makefile.inc` has the content of the lines removed:
```console
[root@5dac5c4dbb17 ccd_andor]# egrep 'BUILD_ROOT|BUILD_DRIVERS|BUILD_LIB|BUILD_INCLUDE' ../../Makefile.inc
BUILD_ROOT = /indigo/build
BUILD_DRIVERS = /indigo/build/drivers
BUILD_LIB = /indigo/build/lib
BUILD_INCLUDE = /indigo/build/include
```

**Review by rumengb — APPROVED** [review]
OK it was my fault! Thank you for the fix!

**Comment by rumengb** [comment]
Actually, this is not enough. We need to generate driver metadata to make indigo_server list it in the drivers property.
I have fixed it!

**Comment by wschoenell** [comment]
great, thank you for taking the time to look into this.
